LabApi/Loader/PluginLoader.cs

```diff
@@ -168,6 +170,24 @@ public static void LoadPlugins(IEnumerable<FileInfo> files)
         }
     }
 
+    private static void InstantiatePlugins(Type[] types, Assembly assembly)
```

Comment:
```suggestion
    private static void InstantiatePlugin(Type[] types, Assembly assembly)
```

See previous comment.